pkg/daemon/daemon.go

```diff
@@ -497,6 +497,9 @@ func (dn *Daemon) applyNodePtpProfile(runID int, nodeProfile *ptpv1.PtpProfile)
 					output.gnss_serial_port = strings.TrimSpace(gnssSerialPort)
 					section.options["ts2phc.nmea_serialport"] = GPSPIPE_SERIALPORT
 				}
+				if _, ok := section.options["leapfile"]; ok || pProcess == ts2phcProcessName { // not required to check process if leapfile is always included
```

Comment:
shouldn't this be if not OK? so 
if leapfile option is not set (!ok) by customer, set default, 
if set by customer (ok), not use default.

Reply:
If leapfile is included in the config or process is ts2phc - override leapfile.
But since it can be included only in ts2phc config, can we just do 
``` 
if  pProcess == ts2phcProcessName 
```